hs/src/Reach/Eval/Core.hs

```diff
@@ -1551,11 +1552,23 @@ evalAsEnvM sv@(lvl, obj) = case obj of
     foldableObjectEnv = map (\m -> (m, delayStdlib $ "Foldable_" <> m <> "1")) foldableMethods
     foldableValueEnv :: [(SLVar, App SLSVal)]
     foldableValueEnv = map (\m -> (m, retStdLib $ "Foldable_" <> m)) foldableMethods
-    tupleValueEnv :: SLObjEnv
-    tupleValueEnv =
+    tupleValueEnv :: SLVal -> SLObjEnv
+    tupleValueEnv tupSlv =
       M.fromList
         [ ("set", delayCall SLPrim_tuple_set)
         , ("length", doCall SLPrim_tuple_length)
+        , ("includes",
+           do
+             -- I would just do `delayCall SLPrim_tuple_includes`, but it fails
+             -- if tested by `SLPrim_is` as a function, which means that something
+             -- like `mytuple.includes` can't be set as a view function.
+             -- Fixing the `SLPrim_is` implementation seems a lot harder than this.
+             foldable_includes <- lookStdlib "Foldable_includes"
+             return $ (lvl, jsClo (srclocOf tupSlv) "tuple_includes"
+               "(v) => f(tup, v)"
+               (M.fromList [("f", SLV_Prim SLPrim_tuple_includes),
+                            ("tup", tupSlv),
+                            ("Foldable_includes", foldable_includes)])))
```

Comment:
Not needed

Reply:
Can you be a bit more specific?  The comment explains why I didn't do the more obvious thing.  So I'm not sure what to do if this closure stuff is not needed.

Reply:
The closure doesn't actually use `Foldable_includes` anymore

Reply:
Oh, I see now that your comment was about a single line... that makes more sense now...

Reply:
I thought the comment was about the whole block displayed there, which I now realize is mostly for context, but doesn't make it clear which line you are referring to.